### ISSUE TITLE
Why navigation using vim mapping doesn't work for some theme?

### ISSUE THREAD

**Issue by TDHTTTT** [roam-unofficial/roam-toolkit#158]
I know it is probably more of a issue for the particular themes, but I am very curious about how navigation using vim actually works. Why the page will scroll down along with the highlighted block by vim mapping and what prevents the page from doing so.

Btw, one example of a theme that does not scroll nicely with vim mapping is Cosmonaut, and I have opened a related [issue](https://github.com/azlen/roam-themes/issues/23) there.

**Comment by tntmarket** [comment]
Are you using a horizontal css theme by any chance?

Unfortunately, Vim mode doesn't work with that right now, but I've been using this patch in the meantime: https://github.com/tntmarket/roam-toolkit/pull/5

https://www.youtube.com/watch?v=4t9cy_A3-QI

**Comment by TDHTTTT** [comment]
Yes I am using Cosmonaut. I checked the patch but does that only cover h/l navigation? How about j/k vertical navigation on the right sidebar?

**Comment by tntmarket** [comment]
Yeah, I currently use a horizontal theme (leyendecker), and both h/l and j/k work for me
